client/my-sites/domains/domain-management/transfer/transfer-to-other-user/index.jsx

```diff
@@ -334,18 +338,25 @@ class TransferOtherUser extends React.Component {
 	}
 
 	filterAvailableUsers( users ) {
-		return users.filter(
-			( user ) =>
-				includes( user.roles, 'administrator' ) &&
-				this.getWpcomUserId( user ) !== this.props.currentUser.ID
-		);
+		return users.filter( ( user ) => this.getWpcomUserId( user ) !== this.props.currentUser.ID );
 	}
 
 	isDataReady() {
 		return this.props.hasSiteDomainsLoaded && ! this.props.isRequestingSiteDomains;
 	}
 }
 
+const withUsers = createHigherOrderComponent(
+	( Wrapped ) => ( props ) => {
+		const siteId = useSelector( getSelectedSiteId );
```

Comment:
The `TransferToOtherUser` component already receives a `selectedSite` prop from above (from `DomainManagementData`) and uses it heavily. The `withUsers` wrapper should use the same source of truth and look at `props.selectedSite.ID` instead of doing its own Redux selection.

Reply:
I believe keeping this as-is makes the component more flexible, and retrieving the selected site ID is cheap on its own. Also, I don't see much value in passing the ID down in this case. 

Do you feel strongly about this?

Reply:
I don't feel strongly about it and it's not a big issue. But I'd personally do the change 🙂 The `withUsers` wrapper should eventually be merged into the `TransferToOtherUser` component when it becomes functional.

Reply:
Well, I don't intend to convert it at this time, who knows, it may remain a `Component` for a while 😉  I guess this is one of those things that we can do eventually, but at this time they don't matter so much.